src/Features/CSharp/Portable/UsePatternMatching/CSharpIsAndCastCheckWithoutNameDiagnosticAnalyzer.cs

```diff
@@ -216,9 +224,9 @@ private void SyntaxNodeAction(SyntaxNodeAnalysisContext context)
 #pragma warning restore RS1030 // Do not invoke Compilation.GetSemanticModel() method within a diagnostic analyzer
         }
 
-        private static SyntaxNode GetContainer(BinaryExpressionSyntax isExpression)
+        private static SyntaxNode? GetContainer(BinaryExpressionSyntax isExpression)
         {
-            for (SyntaxNode current = isExpression; current != null; current = current.Parent)
+            for (SyntaxNode? current = isExpression; current != null; current = current.Parent)
```

Comment:
Shouldn't the CI fail? Is it an IDE0007 bug? or something else?

https://github.com/dotnet/roslyn/blob/cb400a925dc235b505ee2163dc1501d3433d2a12/.editorconfig#L281-L285

Reply:
You can't use var here as it would change the type.

Reply:
Ah yes. Makes sense. Thanks